### DISCUSSION TITLE
<br/> and <hr/> should be <br /> and <hr />

### DISCUSSION THREAD

**Discussion by greg-raven** [jekyll/jekyll#9703] [General]
According to my syntax checkers, there should be a space before the /> to end certain HTML elements. I'm using the Minima theme with Jekyll, so maybe this is a theme issue, but all of the break and horizontal rule elements are written without the space. Is there a reason for this? It seems as though this should be easy to modify.

**Comment by SoumyaK4** [comment]
It doesn't matter really.
The syntax formatters are just built in that way...
Also, these don't need the '/'
It's `<br> and <hr>`

**Comment by greg-raven** [comment]
I understand these tags don't need the closing "/", but as long as it's there, why not doing it the syntactically-correct way?

**Reply by SoumyaK4** [reply]
not including / is the correct syntax bro

**Reply by greg-raven** [reply]
Fine, then remove the "/". Is it really that difficult?

**Comment by mattr-** [comment]
Your suggestion is noted. Fixing this yourself and submitting a PR to the theme is the fastest way to expedite a fix. If that's something that's interesting to you, please do check out the source code for the theme, make a fork of the repo in GitHub, and prepare a PR.

I'll lock this discussion now since there is a difference of opinion about the slashes and where they should be that is no longer constructive.
